pydoclint/utils/visitor_helper.py

```diff
@@ -90,6 +91,49 @@ def checkClassAttributesAgainstClassDocstring(
 
     docArgs: ArgList = doc.attrList
 
+    if allowInlineClassVarDocs:
```

Comment:
Could you abstract this part out into a helper function?

See `checkDocArgsLengthAgainstActualArgs()` and  `checkNameOrderAndTypeHintsOfDocArgsAgainstActualArgs()` below in this function.

This would make this functionality more modular and easier to test.